status-im/status-react
# Communities Join Screens - Implement all permutations of Context Drawer options

fixes: https://github.com/status-im/status-mobile/issues/14464

There are three follow up issues:

corresponding actions will be implemented in a follow up issue. 
denied request options -  as I was not receiving the message from the backend so I need to look into it more.
handle mute community 

TO TEST:
 Go to a community overview page
 
 Click the three dot options to see the bottom sheet.
 
 The options should be different depending on whether the user is joined etc 
 
 This pr does not implement the functionality but just the designs.
 
 
See the designs: https://www.figma.com/file/apCsJHT8J58JXbJD2igtbd/Context-Drawers-for-Mobile?node-id=167%3A107129&t=aRVvvzqko8zHGE59-0
